packages/powerapps-lib/src/schema/processors/__tests__/schema-processes.spec.js

```diff
@@ -101,7 +101,8 @@ describe('the schema processes', () => {
             sdds_osgridreference: '7868962',
             sdds_addressline1: 'Winscombe',
             sdds_county: 'Somerset',
-            sdds_postcode: 'BS21 2XX'
+            sdds_postcode: 'BS21 2XX',
+            sdds_sourceremote: true
```

Comment:
What is this please?

Reply:
We will go through this when we discuss the queue. The tests in this module are concerned with the various processes involved in the transformation of data from our internal format into an ODATA batch query.